### ISSUE TITLE
Can't reproduce the example

### ISSUE THREAD

**Issue by roger-creus** [irahorecka/chrono24#1]
Trying to run the example in the README returns:

`raise NoListingsFoundException("No listings found.") from e. chrono24.exceptions.NoListingsFoundException: No listings found.`

**Comment by irahorecka** [comment]
Hi - thanks for raising this issue. I fixed this issue in version 0.2.8; let me know if there are further issues

**Comment by roger-creus** [comment]
Thank you for your response! Now I can retrieve some results but it eventually crashes as well with

```
File "/opt/anaconda3/envs/chrono/lib/python3.8/site-packages/chrono24/api.py", line 261, in json
    "url": BASE_URL + get_text_html_tag_attr(self.html, "href"),
  File "/opt/anaconda3/envs/chrono/lib/python3.8/site-packages/chrono24/api.py", line 43, in get_text_html_tag_attr
    html_tag_text = html_tag[attr].strip() if html_tag else ""
  File "/opt/anaconda3/envs/chrono/lib/python3.8/site-packages/bs4/element.py", line 1573, in __getitem__
    return self.attrs[key]
KeyError: 'href'
```

**Comment by irahorecka** [comment]
Hey @roger-creus, thanks again for your help. I couldn't reproduce this bug... did you make many calls to chrono24? I'm wondering if they detected a bot and instead returned a warnings page. Aside from this, I released a new version (0.2.9) that addresses a page-turning bug I detected. 

I'd be happy to help you if this issue keeps persisting!

**Comment by irahorecka** [comment]
@roger-creus I was able to reproduce and patch the second bug you mentioned. The rectified logic can be found in v0.2.10. Please let me know if this issue persists.
